packages/fluentui/react-bindings/src/context-selector/useContextSelectors.ts

```diff
@@ -86,13 +87,13 @@ export const useContextSelectors = <
     [value, selected] as const,
   );
 
-  Object.keys(selectors).forEach((key: Properties) => {
-    if (!Object.is(state[1][key], selected[key])) {
-      // schedule re-render
-      // this is safe because it's self contained
-      dispatch(undefined);
-    }
-  });
+  // schedule re-render when selected context is updated
+  const updatedSelectedContext = Object.keys(selectors).find(
+    (key: Properties) => !Object.is(state[1] /* previous { [key]: selector(value) } */[key], selected[key]),
+  );
+  if (updatedSelectedContext !== undefined) {
```

Comment:
```suggestion
  const hasSelectedValuesUpdates = Object.keys(selectors).every(
    (key: Properties) => Object.is(state[1] /* previous { [key]: selector(value) } */[key], selected[key]),
  );
  if (hasSelectedValuesUpdates !== undefined) {
```

_nit_: should be the same, feel free to ignore if there is no sense

Reply:
I think find stops when it found? and every doesn't. I'll change to use `hasSelectedValuesUpdates`

Reply:
> The every method executes the provided callback function once for each element present in the array until it finds the one where callback returns a falsy value. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every